README.md

```diff
@@ -0,0 +1,111 @@
+# @jridgewell/remapping
+
+> Remap sequential sourcemaps through transformations to point at the original source code
+
+Remapping allows you to take the sourcemaps generated through transforming your code and
+"remap" them to the original source locations. Think "my minified code, transformed with babel and
+bundled with webpack", all pointing to the correct location in your original source code.
+
+With remapping, none of your source code transformations need to be aware of the input's sourcemap,
+they only need to generate an output sourcemap. This greatly simplifies building custom
+transformations (think a find-and-replace).
+
+## Installation
+
+```sh
+npm install @jridgewell/remapping
```

Comment:
Is this `@jridgewell` or `@ampproject`?

Reply:
It's `@jridgewell` until I get approval to release it under `@ampproject`.